packages/bot-web-ui/src/components/dashboard/dashboard-component/load-bot-preview/save-modal.tsx

```diff
@@ -0,0 +1,237 @@
+                                            onFocus={e => setCurrentFocus(e.currentTarget.name)}
+                                            onBlur={() => setCurrentFocus(null)}
+                                        />
+                                    )}
+                                </Field>
+                            </div>
+                            <div className='modal__content-row'>
+                                <RadioGroup
+                                    className='radio-group__save-type'
+                                    name='is_local'
+                                    selected={(() => {
+                                        if (is_authorised && !is_local) return save_types.GOOGLE_DRIVE;
+                                        return save_types.LOCAL;
+                                    })()}
+                                    onToggle={() => setFieldValue('is_local', !is_local)}
+                                >
+                                    <RadioGroup.Item
+                                        id='local'
+                                        label={
+                                            <IconRadio
+                                                text={localize('Local')}
+                                                icon={<Icon icon={is_mobile ? 'IcMobile' : 'IcDesktop'} size={48} />}
+                                            />
+                                        }
+                                        value={save_types.LOCAL}
+                                    />
+                                    <RadioGroup.Item
+                                        id='drive'
+                                        label={
+                                            <IconRadio
+                                                text={'Google Drive'}
+                                                icon={<Icon icon={'IcGoogleDrive'} size={48} />}
+                                                google_drive_connected={is_authorised}
+                                                onDriveConnect={onDriveConnect}
+                                            />
+                                        }
+                                        value={save_types.GOOGLE_DRIVE}
+                                        disabled={!is_authorised}
+                                        className={classNames({
+                                            'dc-radio-group__item-disabled': !is_authorised,
+                                        })}
+                                    />
+                                </RadioGroup>
+                            </div>
+                            <>
+                                <Field name='save_as_collection'>
+                                    {({ field }) => (
+                                        <Checkbox
+                                            {...field}
```

Comment:
```suggestion
                                            {...field}
```
nit: the field spread needs to be at the end.

Reply:
done ✅